tests/specs/third-party-compatibility/jspdf.e2e.js

```diff
@@ -1,22 +1,15 @@
-import SpecMatcher from '../../../tools/browser-matcher/spec-matcher.mjs'
+import { reliableUnload } from '../../../tools/browser-matcher/common-matchers.mjs'
 import runTest from './run-test'
 
-const supported = new SpecMatcher()
-  .include('safari')
-  .include('chrome')
-  .include('edge')
-  .include('firefox')
-  .include('android')
-
 describe('jspdf compatibility', () => {
-  withBrowsersMatching(supported)('2.5.1', async () => {
+  withBrowsersMatching(reliableUnload)('2.5.1', async () => {
     await runTest({
       browser,
       testAsset: 'third-party-compatibility/jspdf/2.5.1.html',
       afterLoadCallback: async () => {
         const [errorsResults] = await Promise.all([
           browser.testHandle.expectErrors(10000, true),
-          $('body').click() // Setup expects before interacting with page
+          browser.execute(function () { document.querySelector('body').click() }) // Setup expects before interacting with page
```

Comment:
What was the reason to switch this from the WDIO built-in to executing a function in the DOM?

Reply:
Deprecation warning in IDE.

Reply:
The IDE is treating this as jQuery which it is not. The deprecation warning is wrong.